NEWS.md

```diff
@@ -2,6 +2,9 @@
 
 ## New features
 
+* `use_logo()` now makes the logo a link to the pkgdown site if the project is
```

Comment:
Can you please integrate with the other `use_logo()` changes?

Reply:
Did so, let me know if it looks OK